modules/subworkflows/build_indices.nf

```diff
@@ -25,6 +25,7 @@ workflow BUILD_INDICES{
         known_indels
         pon
         step
+        tools
```

Comment:
What is tools? Can't see it being used

Reply:
https://github.com/nf-core/sarek/blob/a1451dd5910e01cb59e55b529915ee6c8ab8b8a7/modules/subworkflows/build_indices.nf#L47

Reply:
I just got an error because of it, no idea why it didn't register before